### PR TITLE
OCPBUGS#7152: Adds more detail for the agent create command

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-7152

Link to docs preview: https://57042--docspreview.netlify.app/openshift-enterprise/latest/web_console/web-console.html#web-console-overview_web-console

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/web-console-overview.adoc [hunk 1]
```diff
@@ -20,3 +20,8 @@ INFO Login to the console with user: kubeadmin, password: <provided>
 Use those details to log in and access the web console.
 
 For existing clusters that you did not install, you can use `oc whoami --show-console` to see the web console URL.
+
+[IMPORTANT]
+====
+The `dir` parameter specifies the assets directory, which stores the manifest files, the ISO image, and ** auth ** directory. The ** auth ** directory stores the ** kubeadmin-password ** and ** kubeconfig ** files. As a `kubeadmin` user, you can use `kubeconfig` to access the cluster with the following setting: `export KUBECONFIG=<install_directory>/auth/kubeconfig`. Note that the `kubeconfig` is specific to the generated ISO image, so if the `kubeconfig` is set and the `oc` command fails, it is possible that the system did not boot with the generated ISO image. You can use the contents of ** kubeadmin-password ** file to log into the console during the bootstrap as the `core` user to perform debugging.
+====
```
Comment (xenolinux) [inline modules/web-console-overview.adoc:27]: @bfournie Can you please review?

Comment (bfournie) [reply modules/web-console-overview.adoc:27]: This looks fine but should we create a separate bug to add additional troubleshooting info as suggested by Zane in https://github.com/openshift/openshift-docs/pull/55668#discussion_r1105243059

Comment (xenolinux) [reply modules/web-console-overview.adoc:27]: Yes. We can create a separate bug to address that. For now, I will get this PR merged.

File: modules/web-console-overview.adoc [hunk 2]
```diff
@@ -20,3 +20,8 @@ INFO Login to the console with user: kubeadmin, password: <provided>
 Use those details to log in and access the web console.
 
 For existing clusters that you did not install, you can use `oc whoami --show-console` to see the web console URL.
+
+[IMPORTANT]
+====
+The `dir` parameter specifies the assets directory, which stores the manifest files, the ISO image, and ** auth ** directory. The ** auth ** directory stores the ** kubeadmin-password ** and ** kubeconfig ** files. As a `kubeadmin` user, you can use `kubeconfig` to access the cluster with the following setting: `export KUBECONFIG=<install_directory>/auth/kubeconfig`. Note that the `kubeconfig` is specific to the generated ISO image, so if the `kubeconfig` is set and the `oc` command fails, it is possible that the system did not boot with the generated ISO image. You can use the contents of ** kubeadmin-password ** file to log into the console during the bootstrap as the `core` user to perform debugging.
```
Comment (lahinson) [inline modules/web-console-overview.adoc:26]:
```suggestion
The `dir` parameter specifies the `assets` directory, which stores the manifest files, the ISO image, and the `auth` directory. The `auth` directory stores the `kubeadmin-password` and `kubeconfig` files. As a `kubeadmin` user, you can use the `kubeconfig` file to access the cluster with the following setting: `export KUBECONFIG=<install_directory>/auth/kubeconfig`. The `kubeconfig` is specific to the generated ISO image, so if the `kubeconfig` is set and the `oc` command fails, it is possible that the system did not boot with the generated ISO image. To perform debugging, during the bootstrap process, you can log in to the console as the `core` user by using the contents of the `kubeadmin-password` file.
```

- Per the IBM Style guide, directory and file names should be in monospace font. (See https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element).
- I'm thinking that "...you can use `kubeconfig` to access..." should be "...you can use the `kubeconfig` file to access...", but feel free to ignore me if that's not technically accurate.

**Review by lahinson — COMMENTED** [review]
I provided a few suggestions, mostly related to formatting.

File: modules/web-console-overview.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/web-console-overview.adoc]: Addressed

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**